src/components/footer/footer.tsx

```diff
@@ -0,0 +1,106 @@
+import { Link, NavLink } from '@remix-run/react';
+import styles from './footer.module.scss';
+import classNames from 'classnames';
+import { ROUTES } from '~/router/config';
+
+export interface FooterProps {
+    className?: string;
+}
+
+export const Footer = ({ className }: FooterProps) => {
+    const navItemStyle = ({ isActive }: { isActive: boolean }) =>
+        classNames(styles.navItem, {
+            [styles.active]: isActive,
+        });
+
+    return (
+        <footer className={classNames(styles.root, className)}>
+            <section className={styles.navigation}>
+                <nav>
+                    <ul>
+                        <li>
+                            <NavLink
+                                to={ROUTES.products.to('all-products')}
+                                className={navItemStyle}
+                            >
+                                Shop All
```

Comment:
I guess we can fetch and show all categories in footer.
@alisey @tetianaman  wdyt?

Reply:
No, what if there are 50 of them.